src/app/[locale]/viewer/_components/AASOverviewCard.tsx

```diff
@@ -66,39 +61,20 @@ export function AASOverviewCard(props: AASOverviewCardProps) {
     const specificAssetIds = props.aas?.assetInformation?.specificAssetIds as SpecificAssetId[];
     const navigate = useRouter();
     const [productImageUrl, setProductImageUrl] = useState<string | undefined>('');
-    const [registryAasData] = useRegistryAasState();
     const [, setAasState] = useAasState();
+    const [aasOriginUrl] = useAasOriginSourceState();
```

Comment:
these two variables should always be set together, no?
maybe use props instead of the state and set the Origin in the navigate method as well as the actual aas